dstk-web/src/providers/ApolloProvider.tsx

```diff
@@ -0,0 +1,14 @@
+import { ApolloClient, InMemoryCache, ApolloProvider as Provider } from '@apollo/client';
+
+const client = new ApolloClient({
+    uri: 'http://localhost:4000',
```

Comment:
Thoughts on how we want to parameterize this down the road for local & deployed versions?

Reply:
Chef's choice. Either an environment variable or a secret pulled from somewhere (don't know what are plans for deployment are yet).